programs/svm-spoke/src/event.rs

```diff
@@ -121,9 +121,8 @@ pub struct ClaimedRelayerRefund {
     pub refund_address: Pubkey,
 }
 
-// TODO: update the name of this to EmergencyDeletedRootBundle and in EVM.
```

Comment:
we are merging all other SVM PRs to master, but maybe thats ok if we are merging `svm-dev` branch soon